docker-compose-webapp-deployment.yml

```diff
@@ -17,6 +17,10 @@ services:
         MAPBOX_INIT_LATITUDE: ${MAPBOX_INIT_LATITUDE}
         MAPBOX_INIT_LONGITUDE: ${MAPBOX_INIT_LONGITUDE}
         MAPBOX_INIT_ZOOM: ${MAPBOX_INIT_ZOOM}
+        CVIZ_MAPBOX_TOKEN: ${CVIZ_MAPBOX_TOKEN}
```

Comment:
Same as previous, is this needed to be a separate mapbox token than the one we already inject above?

Reply:
Removed this requirement